extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.css

```diff
@@ -70,13 +70,24 @@ amp-story-page[active] .i-amphtml-story-ad-link {
   z-index: 100001 !important;
 }
 
+[dir=rtl] .i-amphtml-ad-overlay-container {
```

Comment:
How is the `rtl` attribute added?

Reply:
Yes. the `i-amphtml-ad-overlay-container` is just the label that contains the 'Ad' icon. I think we do want to flip that, like the other UI elements.

![image](https://user-images.githubusercontent.com/5449100/43733765-b5310288-9983-11e8-9795-84b72566ddce.png)